x/data/module/module.go

```diff
@@ -140,6 +178,6 @@ func (Module) RegisterStoreDecoder(sdr sdk.StoreDecoderRegistry) {
 // WeightedOperations returns all the data module operations with their respective weights.
 // NOTE: This is no longer needed for the modules which uses ADR-33, data module `WeightedOperations`
 // registered in the `x/data/server` package.
-func (Module) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
-	return nil
+func (a Module) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
+	return a.keeper.WeightedOperations(simState)
```

Comment:
I think we can directly call `simulation.WeightedOperations`.

Reply:
https://github.com/regen-network/regen-ledger/pull/1357/commits/2306d18e345b0dd562f52c5fa6e6e33773c53976 👍🏻